test/integration/Region.e2e.test.js

```diff
@@ -45,13 +44,9 @@ describe('Regions', () => {
         // Wait for the empty region layer to be present
         cy.getByTestId('ba-Layer--region');
 
-        // Assert that the region creator does not exist and no annotations are present
-        cy.getByTestId('ba-RegionCreator').should('not.exist');
+        // Assert that no annotations are present
         cy.get('.ba-RegionAnnotation').should('not.exist');
 
-        // Enter region creation mode
```

Comment:
Should we make a comment here or somewhere that we don't need to explicit enter region mode because of the discoverability feature?